shell/platform/darwin/ios/framework/Source/FlutterTextInputPlugin.mm

```diff
@@ -905,7 +905,27 @@ - (void)setTextInputClient:(int)client {
   _hasPlaceholder = NO;
 }
 
+- (UITextInteraction*)textInteraction API_AVAILABLE(ios(13.0)) {
+  if (!_textInteraction) {
+    _textInteraction =
+        [[UITextInteraction textInteractionForMode:UITextInteractionModeEditable] retain];
+    _textInteraction.textInput = self;
```

Comment:
Does `_textInteraction.textInput` need to be nil'd out somewhere to break the retain cycle?

Reply:
https://developer.apple.com/documentation/uikit/uitextinteraction/3255084-textinput?language=objc this is a weak property and the framework is compiled with ARC so I assume it doesn't increase the retain count?

Reply:
👍  Thanks for checking.